recipes/googleapis/all/conanfile.py

```diff
@@ -25,39 +26,42 @@ class GoogleAPIS(ConanFile):
     homepage = "https://github.com/googleapis/googleapis"
     topics = "google", "protos", "api"
     settings = "os", "arch", "compiler", "build_type"
-    generators = "CMakeDeps", "CMakeToolchain"
     options = {
         "shared": [True, False],
-        "fPIC": [True, False]
+        "fPIC": [True, False],
         }
     default_options = {
         "shared": False,
-        "fPIC": True
+        "fPIC": True,
         }
     exports = "helpers.py"
     short_paths = True
 
+    @property
+    def _protobuf_version(self):
+        return "3.21.9"
+
     def export_sources(self):
         copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=os.path.join(self.export_sources_folder, "src"))
         export_conandata_patches(self)
 
-    def source(self):
-        get(self, **self.conan_data["sources"][str(self.version)], destination=self.source_folder, strip_root=True)
-
     def config_options(self):
         if self.settings.os == "Windows":
             self.options.rm_safe("fPIC")
 
-    def layout(self):
-        cmake_layout(self, src_folder="src")
-
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
             self.options["protobuf"].shared = True
 
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires(f"protobuf/{self._protobuf_version}", transitive_headers=True, transitive_libs=True, run=can_run(self))
```

Comment:
Please add  comment for Transitive libs 🙏

Reply:
```suggestion
        # https://github.com/conan-io/conan-center-index/pull/15601#issuecomment-1493086506
        self.requires(f"protobuf/{self._protobuf_version}", transitive_headers=True, transitive_libs=True, run=can_run(self))
```

Reply:
I'll do since conan team seams to be obsessed by this, not sure it's worth the massive time it takes to rebuild...